sycl/plugins/level_zero/CMakeLists.txt

```diff
@@ -24,7 +24,7 @@ if (NOT DEFINED LEVEL_ZERO_LIBRARY OR NOT DEFINED LEVEL_ZERO_INCLUDE_DIR)
   if (WIN32)
     set(LEVEL_ZERO_LOADER_SOURCE_VERSION v1.2.3)
   else()
-    set(LEVEL_ZERO_LOADER_SOURCE_VERSION v1.4.1)
+    set(LEVEL_ZERO_LOADER_SOURCE_VERSION v1.8.8)
   endif()
```

Comment:
Why do we use different version on Windows and non-Windows platforms?

Reply:
Yes, it is ok to merge this.

Reply:
I don't know the answer to that question, but have another one: why did we need to change the Linux version but not the Windows version?

Reply:
IIRC, there was some problem with using newer loader + Windows GPU driver combination. I wonder if this problem still exists, or we can switch back to using the same loader version for all platforms.
@yanfeng3721, could you test v1.8.8 on Windows, please?